network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/focus_area.html

```diff
@@ -1,16 +1,12 @@
 {% load wagtailimages_tags i18n %}
-          
+
 <div class="col-12 col-lg-4 d-block d-sm-flex flex-sm-row flex-lg-column">
     {% image area.interest_icon width-100 class="icon mr-sm-5 align-self-sm-center align-self-lg-start" %}
     <div class="focus-content">
         <h2 class="h3-heading">{{ area.name }}</h2>
         <p>{{ area.description }}</p>
 
-        {# The following code is based on the requirements noted in https://github.com/mozilla/foundation.mozilla.org/issues/4989 #}
-
-        {% if area.page %}
-            <a href="{{ area.page.url }}">{% trans "Learn more →" %}</a>
-        {% endif %}
+        {% if area.page %}<a href="{{ area.page.url }}">{% trans "Learn more →" %}</a>{% endif %}
```

Comment:
```suggestion
        {% if area.page %}<a href="{{ area.page.url }}" class="cta-link">{% trans "Learn more" %}</a>{% endif %}
```

We can also get this updated in the frontend PR for Focus Areas

Reply:
the cta-link class is different than what's requested (18px/bold), so I'll use a custom rule in home.scss